### PR TITLE
Fix spelling errors found by codespell

### PR DESCRIPTION
**Pull request by garrett**

### CODE REVIEW

File: src/Images.jsx [hunk 1]
```diff
@@ -207,7 +207,7 @@ class Images extends React.Component {
 
         const imageRows = filtered.map(id => this.renderRow(this.props.images[id]));
 
-        const interm = this.props.images && Object.keys(this.props.images).some(id => {
+        const interim = this.props.images && Object.keys(this.props.images).some(id => {
```
Comment (garrett) [inline src/Images.jsx:210]: This is a code change. But it's also changed lower in the file where it's referenced.

Comment (martinpitt) [reply src/Images.jsx:210]: Supposedly it was meant to be an abbreviation for "intermediate" (image), but it's fine like that.

**Review by martinpitt — APPROVED** [review]
Thanks!

File: src/Images.jsx (hunk 1 above)
Comment (marusak) [reply src/Images.jsx:210]: correct, but indeed, if this should always jump out when we do codespell, we can use `interim`

**Pull request merged**